src/components/ContentNode/SectionTitle.vue

```diff
@@ -0,0 +1,86 @@
+<!--
+  This source file is part of the Swift.org open source project
+
+  Copyright (c) 2022 Apple Inc. and the Swift project authors
+  Licensed under Apache License v2.0 with Runtime Library Exception
+
+  See https://swift.org/LICENSE.txt for license information
+  See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+-->
+
+<template>
+  <component
+   :id="!isTargetIDE ? id : null"
```

Comment:
Can this just be `anchor` or am I missing something?

Also, I don't think we need to strip out the `id` attribute even if the actual link to that anchor isn't presented in certain situations (like in IDE views).

Reply:
We should strip out the UI element on hover/focus, but the ID here is fine to stay.

Reply:
I left the id without changing it